src/patterns/components/offline-notification/overview.hbs

```diff
@@ -0,0 +1,7 @@
+---
+order: 10
+notes: |
+  In page notification about lack of internet connectivity.
+---
+
+{{{embed "patterns.components.offline-notification.example"}}}
```

Comment:
Not sure what the purpose is of this pattern that only includes the other pattern?